### PR TITLE
change Chat to a submenu, and add only Support and Keybase

### PR DESCRIPTION
**Pull request by sesam**

### CODE REVIEW

File: _includes/navigation.html [hunk 1]
```diff
@@ -36,13 +36,45 @@
...
+            <a href="https://gitter.im/grin_community/design" class="navbar-item">
+              Workgroup: UX & design
+            </a>
+
+            <hr class="navbar-divider">
+
+            <a href="https://gitter.im/grin-pool/Lobby" class="navbar-item">
+              grin-pool support
+            </a>
+            <a href="https://gitter.im/vault713/wallet713" class="navbar-item">
```
Comment (kargakis) [inline _includes/navigation.html]: While I respect a lot vault713's work, I think the grin website should not hint at specific community projects. These are better listed in the wiki page.

Comment (lehnberg) [reply _includes/navigation.html]: Agree that vault713 (nor any other community project for that matter) should be promoted on the official website.

**Comment by sesam** [comment]
@lehnberg I removed wallet713. Did you mean I should remove more of the possible chats?

File: _includes/navigation.html [hunk 2]
```diff
@@ -36,13 +36,42 @@
...
+            <a href="https://gitter.im/grin_community/dev" class="navbar-item">
+              Workgroup: Development
+            </a>
+            <a href="https://gitter.im/grin_community/design" class="navbar-item">
+              Workgroup: UX & design
+            </a>
+
+            <hr class="navbar-divider">
+
+            <a href="https://gitter.im/grin-pool/Lobby" class="navbar-item">
```
Comment (kargakis) [inline _includes/navigation.html]: This should also be removed.

File: _includes/navigation.html [hunk 3]
```diff
@@ -36,13 +36,42 @@
...
+              grin-pool support
+            </a>
+
+            <hr class="navbar-divider">
+
+            <a href="https://gitter.im/grinexchange/community" class="navbar-item">
```
Comment (kargakis) [inline _includes/navigation.html]: Unclear about this.

**Comment by kargakis** [comment]
The `grin_community` channels lgtm, not sure about `grinexchange`, `grin-pool` similarly to `wallet713` should be removed.

**Pull request merged**